bigquery/storage/managedwriter/options.go

```diff
@@ -61,14 +61,14 @@ type writerClientOption interface {
 	ApplyWriterOpt(*writerClientConfig)
 }
 
-// EnableMultiplexing is an EXPERIMENTAL option that enables connection sharing
+// WithMultiplexing is an EXPERIMENTAL option that controls connection sharing
 // when instantiating the Client.  Only writes to default streams can leverage the
 // multiplex pool.  Internally, the client maintains a pool of connections per BigQuery
 // destination region, and will grow the pool to it's maximum allowed size if there's
 // sufficient traffic on the shared connection(s).
 //
 // This ClientOption is EXPERIMENTAL and subject to change.
-func EnableMultiplexing(enable bool) option.ClientOption {
+func WithMultiplexing(enable bool) option.ClientOption {
```

Comment:
This one I think is fine to not use the With and instead pass nothing. The presence of the option seems like a logical true if passed. But I will leave it up to you

Reply:
Decided to do `WithMultiplexing()` here without argument(s).